app/controllers/people/index.js

```diff
@@ -17,6 +17,14 @@ export default class PeopleIndexController extends Controller {
 
   @action
   search() {
-    this.router.refresh();
+    if (this.page > 0) {
+      this.resetPagination(); // updating `page` will refresh the model
```

Comment:
This looks a bit weird, but updating the sort query param already reloads the model so refresh isn't needed here. Not sure how to make that more obvious without a comment.